generators/fastlane-setup/templates/fastlane/Fastfile

```diff
@@ -257,6 +271,15 @@ platform :android do
     )
   end
 
+  lane :upload_to_playstore_alpha do |options|
+    supply(
+      package_name: ENV['GRADLE_APP_IDENTIFIER'],
+      track: 'alpha',
```

Comment:
At BAM, we do: 

* internal : developers (no client)
* alpha : PO and stakeholder, + additional client testers
* beta : other testers

Can you change the track to `internal`?